services/QuillLMS/app/controllers/teachers/classrooms_controller.rb

```diff
@@ -145,8 +154,8 @@ def transfer_ownership
           SegmentIo::BackgroundEvents::TRANSFER_OWNERSHIP,
 { properties: { new_owner_id: requested_new_owner_id } }
       )
-    rescue
-      return render json: { error: 'Please ensure this teacher is a co-teacher before transferring ownership.' }, status: 401
+    rescue => e
+      return render json: { error: "Please ensure this teacher is a co-teacher before transferring ownership. #{e}" }, status: 401
```

Comment:
What condition is this `rescue` trying to save us from?  Since we should be redirecting users if they are not the owner before they even get into this block, does that mean that the error text here is likely to be inaccurate?